theano/misc/check_multi_gpu.py

```diff
@@ -0,0 +1,50 @@
+#! /usr/bin/env python
+import time
+
+import numpy
+
+import pygpu
+import theano
+from theano.sandbox.gpuarray.type import (reg_context,
+                                          gpuarray_shared_constructor as shared)
+from theano.sandbox.gpuarray.blas import gpu_dot22
+
+def main(dev1, dev2):
+    ctx1 = pygpu.init(dev1)
+    ctx2 = pygpu.init(dev2)
+    print "ctx1", ctx1.devname
+    print "ctx2", ctx2.devname
+
+    reg_context('gpu1', ctx1, dev1)
```

Comment:
gpuN have a meaning as a device. I think it is a bad habit to reuse that pattern. Other people will reuse that and not have an understanding as good as us. what about reusing the string ctx1, ctx2?